src/components/Articles.jsx

```diff
@@ -0,0 +1,44 @@
+import { useEffect, useState } from "react"
+import Articlecard from "./Articlecard"
+
+const Articles = () => {
+    const [articles, setArticles] = useState([])
+    const [loading, setLoading] = useState(false)
+    const [error, setError] = useState(null)
+
+    useEffect(() => {
+        setLoading(true)
+        fetch('https://my-nc-news-ew4b.onrender.com/api/articles')
+        .then((response) =>{
+            return response.json()
+        })
+        .then(({ articles }) =>{
+            setArticles(articles)
+            setLoading(false)
+        })
+        .catch((error) => {
+            setLoading(false)
+            setError(error)
+        })
+    }, [])
+
```

Comment:
Nice, but I would recommend also looking into axios for an alternative way to do this as well as extracting out some of this logic into a function. (Have a read of the notes on axios for this)